app/views/static_pages/contributing.html.erb

```diff
@@ -1,68 +1,107 @@
+          <div class="card-deck w-75">
+            <div class="card mb-5 mb-md-0">
+              <div class="card-block">
+                <h4 class="card-title camel font-weight-bold">Contribute to the Curriculum</h4>
+                <p class="card-text">Anyone with any experience level can contribute to the curriculum. Students in particular are encouraged to do this, especially if you are just starting out!</p>
+                <p class="card-text">Tasks include:</p>
+                <p class="card-text">
+                  <ul class="dashed">
+                    <li>Adding your own solutions to lessons.</li>
+                    <li>Updating individual lessons with resources.</li>
+                    <li>Writing new lessons or updating any that are unfinished.</li>
+                  </ul>
+                </p>
+              </div>
+                <div class="mx-auto card-base">
+                  <%= link_to 'Go to Github curriculum', github_link('curriculum'), class: 'btn btn-contribute', target: '_blank' %>
+                </div>
+            </div>
+            <div class="card">
+              <div class="card-block">
+                <h4 class="card-title camel font-weight-bold">Contribute to the Main Site</h4>
+                <p class="card-text">
+                  The Odin Project is a Ruby on Rails application. As with any Open Source project, there is a constant backlog of outstanding issues and new features. Anyone is welcome to join in and start working on any one of these tickets - the more the merrier!
+                </p>
+                <p class="card-text">Tasks include:</p>
+                <p class="card-text">
+                  <ul class="dashed">
+                    <li>Viewing our Trello Board where we keep a backlog fo outstanding issues and new features.</li>
+                    <li>Picking issues or features to work on.</li>
+                  </ul>
+                </p>
+              </div>
+                <div class="mx-auto card-base">
+                  <%= link_to 'Go to Github main site', github_link('theodinproject'), class: 'btn btn-contribute', target: '_blank' %>
+                </div>
+            </div>
+          </div>
+        </div>
+        <br/>
+        <div class="row">
+          <div class="col-md-8 offset-md-2 mb-5">
+            <p class="text-muted">
+              Instructions on setting up your local development environment can be found in the <%= link_to 'GitHub repo', github_link, target: '__blank' %>.
+            </p>
+            <p class="text-muted">
+              <em>NOTE: When choosing an issue to work on, please check out the <%= link_to '#contributing', chat_link, class: 'gitter-chat-link', target: '_blank' %> channel on our Gitter chat, and let everyone know what you are working on.  This is important, as somebody else may have already done some work on the issue - in which case they would probably love to have your help!</em>
```

Comment:
The contributing link should be pointing to https://gitter.im/TheOdinProject/Contributing